client.go

```diff
@@ -307,6 +319,13 @@ func (client *NativeClient) Session() (*ssh.Session, *ssh.Client, *ssh.Client, e
 	return session, proxy, conn, nil
 }
 
+//RemoteOutput returns the of the command run proxied through the host the remote host. The same credentials and timeout are used.  The remote key must be in the home directory
+func (client *NativeClient) RemoteOutput(remoteHost, command string) (string, error) {
+	timeout := fmt.Sprintf("ConnectTimeout=%v", client.HostConfig.Timeout.Seconds())
+	sshCmd := fmt.Sprintf("ssh -o %s -o %s -o %s -o %s -i %s %s@%s \"%s\"", SSHOpts[0], SSHOpts[1], SSHOpts[2], timeout, client.RemoteKey, client.HostConfig.User, remoteHost, command)
```

Comment:
To avoid having to match this command with the length of the SSHOpts array, I suggest:

```
args := []string{"ssh"}
for _, op := range SSHOpts {
        args = append(args, "-o", op)
}
args = append(args, "-o", timeout, "-i", client.RemoteKey, client.HostConfig.User + "@" + remoteHost, command)
return client.Output(strings.Join(args, " "))
```